composer.json

```diff
@@ -17,13 +17,13 @@
     "Tao",
     "oat",
     "open source",
-    "release 3.5",
+    "release 4.0",
     "computer-based-assessment"
   ],
   "homepage": "http://www.taotesting.com",
   "minimum-stability": "stable",
   "prefer-stable": true,
   "require": {
-    "oat-sa/tao-community": "2021.11.8"
+    "oat-sa/tao-community": "2022.11.6"
```

Comment:
Shall we use this one? https://github.com/oat-sa/tao-community/tree/2022.11.7

Reply:
Or maybe even https://github.com/oat-sa/tao-community/tree/v2022.11.9.

It has that latest fix by Focaloid, which looked important.